tasks/Task.cpp

```diff
@@ -906,6 +906,12 @@ void Task::acquisitionConfiguration(Arena::IDevice& device)
         ptp_sync_frame_rate->SetValue(config.frame_rate);
     }
     else {
+        double frame_timeout = _image_config.get().frame_timeout.toSeconds();
+        double seconds_per_frame = 1 / config.frame_rate;
+        if (frame_timeout < seconds_per_frame) {
+            exception(FPS_SURPASSES_FRAME_TIMEOUT);
+            return;
+        }
```

Comment:
This actually will let the configuration continue.

throw an exception instead.